docs/docs/contributing/code-of-conduct.md

```diff
@@ -1,4 +1,4 @@
-# Contributor Covenant Code of Conduct
+# Code of Conduct
```

Comment:
Was thinking about how that file makes sense in both places, still think it's worth to somehow mirror their content rather than just link one to another. 

Think that's doable without chores? Judging this by the number of updates that file gets and some negative experience running parallel build systems (copy stuff from one place to another as a npm pre or git hook)